algobot/traders/trader.py

```diff
@@ -246,9 +246,9 @@ def get_stop_loss_strategy_string(self) -> str:
         Returns stop loss strategy in string format, instead of integer enum.
         :return: Stop loss strategy in string format.
         """
-        if self.lossStrategy == STOP:
+        if self.lossStrategy == OrderType.STOP:
```

Comment:
We should probably deprecate this function and add a kwarg for suffix in the to_str method